File: EndGate/EndGate.Core.Client.JS/Collision/CollisionData.ts
/// <reference path="../Assets/Vectors/Vector2d.ts" />
/// <reference path="Collidable.ts" />

module EndGate.Core.Collision {

    export class CollisionData {
        public At: Assets.Vector2d;
        public With: Collidable;

        constructor(at: Assets.Vector2d, w: Collidable) {
            this.At = at;
            this.With = w;
        }
    }

}

File: EndGate/EndGate.Core.Net/Collision/CollisionData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using EndGate.Core.Net.Assets;

namespace EndGate.Core.Net.Collision
{
    public class CollisionData
    {
        public CollisionData(Vector2d at, Collidable with)
        {
            At = at;
            With = with;
        }

        public Vector2d At { get; private set; }
        public Collidable With { get; private set; }
    }
}
